en/assembly/quick_start_pixhawk4_mini.md

```diff
@@ -0,0 +1,134 @@
+# *Pixhawk 4 Mini* Wiring Quick Start
+
+This quick start guide shows how to power the [*Pixhawk<sup>&reg;</sup> 4 Mini*](../flight_controller/pixhawk4_mini.md) flight controller and connect its most important peripherals.
+
+<img src="../../assets/flight_controller/pixhawk4mini/pixhawk4mini_iso_1.png" width="350px" title="Pixhawk4 Image" />
+
+## Wiring Chart Overview
+
+The image below shows where to connect the most important sensors and peripherals (except for motors and servos).
+
+![*Pixhawk 4 Mini* Wiring Overview](../../assets/flight_controller/pixhawk4mini/pixhawk4mini_wiring_overview.png)
```

Comment:
The text in this image references this page in URL.
![image](https://user-images.githubusercontent.com/5368500/45522396-20898c80-b806-11e8-9f61-a7c147e82b4c.png)

Reply:
The CAP 4,3,2,1 do not appear to be documented anywhere either - did I miss this?

Reply:
Just on the pinouts pdf. They were not really documented on the Pixhawk 4 aside from that either. Do you think it would be best to put additional information on those outside PDF of pinouts?

Reply:
So I had one of these images for the online docs and one for print. I forgot to take out that url.

Reply:
No worries. 

> Just on the pinouts pdf. They were not really documented on the Pixhawk 4 aside from that either. Do you think it would be best to put additional information on those outside PDF of pinouts?

Yes, probably in the other .md doc (not quickstart, which is only for quick start). Ideally enough for someone to do anything they want with the port, but minimally enough to understand what it is for - and the moment I have no idea why this is here.